src/test/resources/localstack/setup-aws.sh

```diff
@@ -9,5 +9,7 @@ export PAGER=
 # Create the bucket
 aws s3 --endpoint-url=http://localhost:4566 --region eu-west-2 ls s3://local-644707540a8083b7b15a77f51641f632 || aws --endpoint-url=http://localhost:4566 --region=eu-west-2 s3 mb s3://local-644707540a8083b7b15a77f51641f632
 
-echo "S3 Configured with cpg-bucket"
+aws --endpoint-url=http://localhost:4566 sns create-topic --name court-cases-topic.fifo --attributes '{"FifoTopic":"true", "ContentBasedDeduplication":"true"}'
```

Comment:
this will be done automatically on application startup, why is it in here as well?